cloudfront-dist/deploy.sh

```diff
@@ -1,10 +1,12 @@
-#!/bin/sh -eu
-
+#!/bin/sh
+set -eu
 # WORK ONLY with the docker images !
-# Require TARGET_ENVIRONMENT=test 
-cd server && npm run gen-lambda && cd .. && \
-    rm ./static/index.html && \
-    # Deploy, move to deployement dir in order to have access to the app in cdk.json 
-    cd cloudfront-dist/deployment && \
-    echo "run cdk deploy --require-approval never $STACK_PREFIX_NAME-$FRONT_APP_NAME" && \
-    cdk deploy --require-approval never $STACK_PREFIX_NAME-$FRONT_APP_NAME
\ No newline at end of file
+# Require TARGET_ENVIRONMENT=test
+cd server
```

Comment:
i prefer the model line, if any command fails it stops

Reply:
The `set -eu` instruction do the job.